PL2/PL2-shared/src/main/java/nl/tudelft/pl2016gr2/model/OriginalGraph.java

```diff
@@ -17,10 +17,14 @@ public OriginalGraph() {
   /**
    * Print a string representation of this graph.
    */
-  public void print() {
+  @Override
+  public String toString() {
+    StringBuilder sb = new StringBuilder();
     for (Node node : nodes.values()) {
-      System.out.println(node);
+      sb.append(node);
+      sb.append("\n");
```

Comment:
@FarisElghlan not to be very nitpicky, but `sb.append(node + '\n')` would just be a concatenation inside the argument, wouldn't it?

@cbilstra for characters that would normally appear on a newline I would agree, but the newline character in text only appears at the end of a line (so directly after node). If you had another line of text appended, then that would as you say, fall onto the next line.
```
sb.append(node).append('\n');
sb.append("next line of text");
```